dep_dqa.py

```diff
@@ -164,7 +164,7 @@ def dep_dqa(instrObj, tpx=0):
     tablesDir = instrObj.metadataTablesDir
     ymd = utDate.replace('-', '')
     metaOutFile =  dirs['lev0'] + '/' + ymd + '.metadata.table'
-    keywordsDefFile = tablesDir + '/keywords.format.' + instr
+    keywordsDefFile = tablesDir + f'/KOA_{instr.upper()}_Keyword_Table.txt'
```

Comment:
Have we decided on the checkout location for the KeywordTables repo?  I'm thinking vm-koaserver5:/kroot/archive/KeywordTables/

Reply:
We have not. I will try putting them there.

Reply:
No go. Perhaps someone else has writing privileges to /kroot/archive?